src/components/Button/Button.scss

```diff
@@ -7,6 +7,17 @@ $slim-vertical-padding: ($slim-min-height - line-height(body) - rem(2px)) / 2;
 $large-min-height: rem(44px);
 $large-vertical-padding: ($large-min-height - line-height(body) - rem(2px)) / 2;
 $spinner-size: rem(20px);
+$pressed-border-color: rgb(184, 195, 205);
+$pressed-hover-background: rgb(217, 222, 228);
+$pressed-active-background: rgb(211, 217, 223);
+$primary-pressed-lightest: rgb(63, 78, 174);
+$primary-pressed-light: rgb(60, 75, 167);
+$primary-pressed-dark: rgb(16, 23, 60);
+$primary-pressed-darkest: rgb(11, 15, 39);
+$destructive-pressed-lightest: rgb(176, 43, 19);
+$destructive-pressed-light: rgb(167, 41, 18);
+$destructive-pressed-dark: rgb(117, 4, 10);
+$destructive-pressed-darkest: rgb(93, 3, 8);
```

Comment:
I've got some concerns on the impact all these snowflake colors will have on the color system work, and the move to client-side theming. Some context on how these colors were generated would give us a stronger ability to upgrade this to consume the color system. I actually think this was better with the `darken` and `rgba` functions, since we know where the colors started and how they got to the final state. That feels easier to systematically reproduce.

@tmlayton, what do you think about going back to the way it was before, given what I said above?

Reply:
Either that, or leave comments on each of these colors to show how they were achieved using programmatic darkening of our color tokens.

Reply:
I agree with Dan here. If there were achieved using the `darken` functions it might be best to leave it. It does make it clear on how to reproduce across themes.

Reply:
Added comments on how the colors were achieved